### PR TITLE
Refactor Calendar Feature to Seribase

### PR DESCRIPTION
**Pull request by rfaturriza**

### CODE REVIEW

File: lib/features/calendar/data/models/calendar_model.dart [hunk 1]
```diff
@@ -72,6 +74,31 @@ class CalendarModel {
     );
   }
 
+  factory CalendarModel.fromSeribase(Map<String, dynamic>? data) {
```
Comment (faisaladnanpeltops) [inline lib/features/calendar/data/models/calendar_model.dart:77]: is the factory _fromSeribase_ being used from the repository or presentation class?

**Comment by faisaladnanpeltops** [comment]
https://github.com/user-attachments/assets/542bd9da-8a81-400b-ae5f-4c974ffba448

Bug:
- Calendar entries are missing after creation

File: lib/features/calendar/data/models/calendar_model.dart (hunk 1 above)
Comment (rfaturriza) [reply lib/features/calendar/data/models/calendar_model.dart:77]: repository

**Comment by rfaturriza** [comment]
> kalendar.mp4 
> Bug:
> 
> * Calendar entries are missing after creation

Noted

**Review by faisaladnanpeltops — APPROVED** [review]

**Pull request merged**